### ISSUE TITLE
accessibility issue 33 (medium) -make tombstone font black instead of gray

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#248]
All of the light gray text used within the image details block has insufficient color contrast.  The contrast should be at least 4.5:1.

**Comment by yulgit1** [comment]
resolved: https://github.com/ycba-cia/blacklight-collections2/commit/9e7789494c5db2262df4e57aefe6e58260f1548b

**Comment by yulgit1** [comment]
applies to issue 36 too:

The color contrast is insufficient for the details in the light gray color in the list view of the search results.  The color contrast should be 4:5:1 for the text color on the background color.
